.github/actions/setup-deps/action.yml

```diff
@@ -0,0 +1,64 @@
+name: "Turborepo CI Setup"
+description: "Sets up everything for CI"
+inputs:
+  github-token:
+    description: "Github token. You can pass secrets.GITHUB_TOKEN"
+    required: true
+  setup-node:
+    description: "Set to false to prevent Node setup steps"
+    required: false
+    default: true
+  go-version:
+    description: "Change Go version"
+    required: false
+    default: "1.18"
+runs:
+  using: "composite"
+  steps:
+    - name: Set up Go
+      uses: actions/setup-go@v3
+      with:
+        go-version: ${{ inputs.go-version }}
+        cache: true
+        cache-dependency-path: cli/go.sum
```

Comment:
It feels super-weird to hardcode this here as it's an application concern, not a software concern. (super not reusable, happens to work anyway).

But throwing it as an arg is also silly when that just makes consumption unnecessarily verbose.

Reply:
🤷🏾 , perhaps a future discussion PR? Not sure I understand your concern, tbh.

Reply:
Just feels weird, not really intended to be actionable.